src/usdb_syncer/constants.py

```diff
@@ -1,11 +1,13 @@
 """Constants"""
 
 import re
+import sys
 
 # set in the release workflow
 VERSION = "dev"
 COMMIT_HASH = "dev"
 SHORT_COMMIT_HASH = COMMIT_HASH[:7]
+NUL = "NUL" if sys.platform == "win32" else "/dev/null"
```

Comment:
Does not fit the block's comment. Also the variable name could be a bit more descriptive.

Reply:
Just found out that there is `os.devnull`, which makes things cleaner.